api/pkg/cli/cmd/info/info.go

```diff
@@ -139,8 +139,70 @@ func (opts *options) run() error {
 	}
 
 	hubClient := opts.cli.Hub()
+
+	selectOption := so.Options{}
+
+	if opts.from == "" {
+		// Get all Catalogs
+		catalog, err := hubClient.GetCatalogsList()
+		if err != nil {
+			return err
+		}
+		if len(catalog) == 1 {
+			opts.from = catalog[0]
+		} else {
+			// Ask the catalog
+			err = selectOption.Ask("catalog", catalog)
+			if err != nil {
+				return err
+			}
+			opts.from = selectOption.Catalog
```

Comment:
can you move this to a seperate func

Reply:
Any reason to move this into a seperate function ??

Reply:
readability :D

Reply:
Updated the pr!